### DISCUSSION TITLE
Can we separate Magic Mouse from other Mouses, or recognize it as a Trackpad, or allow per-device reversals?

### DISCUSSION THREAD

**Discussion by cmsflash** [pilotmoon/Scroll-Reverser#182] [Ideas]
The main reason I am looking for a scroll reverser is that:
- On a trackpad, I feel like I'm moving the page I'm reading like a piece of paper, so if my fingertip moves up, the page should go up;
- On a mouse with a scrolling wheel, I feel like I'm moving the page I'm reading through the wheel, which would reverse the direction, so when my fingertip moves up, the page should move down;

By this standard, I feel like the Magic Mouse should belong to the trackpad category, since it does not have a scrolling wheel, instead I'm gesturing against a surface directly, similar to a trackpad.

**Comment by pilotmoon** [comment]
I agree this option would be a useful addition to Scroll Reverser.

For now, if you only want to reverse scroll wheels, you could use one of the other apps (listed in the Alternatives section on Scroll Reverser home page), none of which reverse the magic mouse. So they kind of have the feature you want by default. Most of the magic of SR is to be able to tell that a Magic Mouse is a mouse The other apps are less complex precisely because they do not try to do that.

**Reply by cmsflash** [reply]
Ic. TY for the explanation and pointer!

**Reply by In0chi** [reply]
I hope I’m not upsetting the developers here, but I wanted to mention another small dev team over at [https://multitouch.app/](https://multitouch.app/). They produce an app that allows for custom gestures and controls specifically for the Magic Trackpad and Mouse.
